kpi/views/v2/asset.py

```diff
@@ -722,6 +722,17 @@ def get_serializer_context(self):
 
             context_['children_count_per_asset'] = children_count_per_asset
 
+            # 5) Get organization per asset
+            assets = (
+                Asset.objects.filter(id__in=asset_ids)
+                .prefetch_related('owner__organizations_organization')
+            )
+            organization_by_asset = defaultdict(dict)
+            for asset in assets:
+                organization = getattr(asset.owner, 'organization', None)
+                organization_by_asset[asset.id] = organization
+            context_['organization_by_asset'] = organization_by_asset
+
```

Comment:
Do we have the same optimization for `/api/v2/organizations/<organization_id>/assets/`?